tools/add_triage_signature.py

```diff
@@ -1394,7 +1394,13 @@ def __init__(self, *args, **kwargs):
     def _process_ticket(self, url, issue_key):
         md = get_metadata_json(url)
         cluster_id = md["cluster"]["id"]
-        if md["cluster"]["logs_info"] in ("timeout", "completed"):
+        # temporary workaround till requested status is set only when the controller is
+        # already up and running
+        cluster = md["cluster"]
+        hosts = cluster["hosts"]
+        some_done = any(host["progress"] == "Done" for host in hosts)
+        ##
+        if some_done and md["cluster"]["logs_info"] in ("timeout", "completed", "collecting"):
```

Comment:
Why collecting?
if it's still collecting we might still get it

Reply:
We run it on failed clusters so that might be an edge case where the triage ticket is scrapped before the must-gather has finished (by mistake). This signature is designed to find such strange edge cases

Reply:
That cluster logs will keep getting updated until the logs_info reach `timeout` or `collected`. 
see https://github.com/openshift/assisted-test-infra/pull/1974/files

Reply:
collecting removed since the signature can run any time during the cluster lifecycle and may report inaccurate status for collecting state that may eventually result in successful log scrapping.